# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../libtomcrypt/pdflatex.patch
# Copyright (C) 2006 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./makefile.orig	2006-12-02 00:44:00.000000000 -0300
+++ ./makefile	2006-12-02 00:51:47.000000000 -0300
@@ -341,8 +341,7 @@
 	latex crypt > /dev/null
 	makeindex crypt.idx > /dev/null
 	perl fixupind.pl
-	latex crypt > /dev/null
-	dvipdf crypt
+	pdflatex crypt > /dev/null
 	mv -ivf crypt.pdf doc/crypt.pdf
 	rm -f $(LEFTOVERS)
 
